ReactiveCocoa.podspec

```diff
@@ -60,7 +60,10 @@ Pod::Spec.new do |s|
     ]
     ss.osx.framework = 'AppKit'
 
-    ss.tvos.source_files = '**/ReactiveCocoa.h'
+    ss.tvos.source_files = [
+      '**/ReactiveCocoa.h',
+      'ReactiveCocoa/**/*{UIButton,UICollectionReusableView,UIControl,UIGestureRecognizer,UISegmentedControl,UITableViewCell,UITableViewHeaderFooterView,UIText}*'
```

Comment:
What are some examples of this? 2 of the other 3 places are in the project file.